oryx-app-mllib/src/test/java/com/cloudera/oryx/app/mllib/kmeans/AbstractKMeansEvalIT.java

```diff
@@ -38,6 +46,20 @@
           {-2.0, 0.0}, {-0.5, -1.0}, {-0.5, 1.0}
       });
   protected static final DistanceFn<double[]> DISTANCE_FN = new SquaredDistanceFn();
+  private JavaSparkContext javaSparkContext;
+  private static JavaRDD<double[]> pointsRDD;
```

Comment:
This is also supposed to be non-static or it's going to clash with other tests running at the same time